src/options.ts

```diff
@@ -147,6 +147,13 @@ export async function loadOptions (userConfig: NitroConfig = {}): Promise<NitroO
   //     .map(i => new RegExp(`(^|\\/)${escapeRE(i.split('node_modules/').pop())}(\\/|$)(?!node_modules\\/)`))
   // ]
 
+  if (options.autoImport) {
+    options.autoImport.exclude = options.autoImport.exclude || [/[\\/]node_modules[\\/]/, /[\\/]\.git[\\/]/]
```

Comment:
Default config merging strategy is joining (not override) also we are already pushing buildDir. I think we can add node_modules and .git to the defaults as well.

Reply:
Actually, are we sure it doesn't override?

https://github.com/unjs/unimport/blob/78e8d7416b2ea723167777cc3a1175109e850358/src/unplugin.ts#L26-L29

Reply:
Yes, I'm aware unimport does override. Nitro/Nuxt options extend in the other hands. Also good idea for duplicating defaults for predictable behavior 👍🏼

Reply:
Oh, so you were suggesting we always push `.git` and `node_modules` as well?

Reply:
Just checking 😉